src/components/Filters.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+class Filters extends React.Component {
+  onCreateClick = () => {
+    this.props.history.push("/create");
+  };
+
+  render() {
+    return (
+      <div>
+        <input autoFocus placeholder="Search event" />
+        <select>
+          <option>Select city</option>
+          <option>Cracow</option>
+          <option>Wroclaw</option>
```

Comment:
Please, create an array of those names and map through them. Do this also for categories.